const fail = (error) => {
  throw new Error(error);
};

export default fail;
